File: dashboard/src/types/runs.ts
export type PipelineRun = {
  runId: string
  status: string
  startedAt: string
  completedAt: string
  duration: string
  ec2Findings: number
  s3Findings: number
  blocked: number
  phase3ParseErrors: number
  totalSavings: number
}

File: dashboard_backend/app/schemas/runs.py
from pydantic import BaseModel


class RunResponse(BaseModel):
    runId: str
    status: str
    startedAt: str
    completedAt: str
    duration: str
    ec2Findings: int
    s3Findings: int
    blocked: int
    phase3ParseErrors: int
    totalSavings: float
